lib/compiler_rt/count0bits.zig

```diff
@@ -73,7 +73,7 @@ fn __clzsi2_thumb1() callconv(.Naked) void {
         \\ subs r1, #4
         \\ movs r0, r2
         \\ 1:
-        \\ ldr r3, .lut
+        \\ ldr r3, =.lut
```

Comment:
```suggestion
        \\ adr r3, .lut
```

I *think* this'll do what we want: Load the address of `.lut` PC-relative instead of with a relocation. Please do test it though; I'm just going off assembler documentation.

Reply:
Tested it in a debugger and it appears to load the correct address.